docs/content/Installation/postgres-operator-installer/ansible-pgo-installer.md

```diff
@@ -90,6 +88,38 @@ $PGOROOT/installers/pgo-installer/ansible/main.yml
 | `install-container` | Uses the `pgo-installer` image to install the PostgreSQL Operator. |
 | `update-container` | Uses the `pgo-installer` image to update the PostgreSQL Operator. |
 | `uninstall-container` | Uses the `pgo-installer` image to uninstall the PostgreSQL Operator. |
+| `setup-client` | Runs setup steps to use the `pgo` client locally |
 | `clean` | This option can be added to the `install-container`, `update-container`, and `uninstall-container` tags to delete the job after it completes. |
 | `clean-all` | The `namespace` and `clusterrolebinding` will be deleted if they exist. |
 
+### PGO Client
+#### Using the PGO Client Locally
```

Comment:
These sections and headers should match what is being used for the installer container for consistency.